src/c++/library/http_client.cc

```diff
@@ -26,17 +26,19 @@
 
 // Include this first to make sure we are a friend of common classes.
 #define TRITON_INFERENCE_SERVER_CLIENT_CLASS InferenceServerHttpClient
-#include "common.h"
+#include "http_client.h"
 
 #include <curl/curl.h>
+
 #include <atomic>
 #include <climits>
 #include <cstdint>
 #include <deque>
 #include <iostream>
 #include <string>
 #include <utility>
-#include "http_client.h"
+
```

Comment:
same here

Reply:
Auto-formatter. Do you want me to save without formatting?

Reply:
There is no standard currently for what formatter to use. And we have no mechanism currently for codifying proposed standards.

Reply:
Undid auto-formatting changes 👍